tests/integration/devfile/cmd_devfile_list_test.go

```diff
@@ -4,6 +4,10 @@ import (
 	"fmt"
 	"path/filepath"
 
+	"github.com/tidwall/gjson"
```

Comment:
We decided to refactor the tests file whenever some PR modifies it because it is not possible to otherwise refactor our entire test suite as proposed in #4770. Do you want to do this in this PR or in a separate PR right after this PR?